roboteam_ai/src/control/ballHandling/BallHandlePosControl.cpp

```diff
@@ -298,7 +321,18 @@ RobotCommand BallHandlePosControl::goToIdleBall(
     else {
         numTreesTarget = ball->pos + ballToTarget.stretchToLength(maxBallDistance);
     }
-    return NumTreePosControl::getRobotCommand(robot, numTreesTarget);
+    auto robotCommand = NumTreePosControl::getRobotCommand(robot, numTreesTarget);
+    if ((ball->pos - robot->pos).length() < maxBallDistance*1.5) {
+        robotCommand.angle = (ball->pos - robot->pos).toAngle();
+    }
+    return robotCommand;
+}
+
+RobotCommand BallHandlePosControl::updatePID(PID &xpid, PID &ypid, const RobotCommand &robotCommand) {
```

Comment:
This is not updating the PID values as the name implies but rather executing it.

Reply:
fixed, renamed